src/ripple/protocol/impl/SecretKey.cpp

```diff
@@ -191,7 +191,7 @@ class Generator
         auto gsk = [this, tweak = calculateTweak(ordinal)]() {
             auto rpk = root_;
 
-            if (secp256k1_ec_privkey_tweak_add(
+            if (secp256k1_ec_seckey_tweak_add(
```

Comment:
Note for myself: These functions are equivalent, but the `privkey` version is deprecated in favor of `seckey`.